python/ql/test/query-tests/Expressions/general/UseofApply.expected

```diff
@@ -0,0 +1 @@
+| expressions_test.py:270:3:270:17 | ControlFlowNode for apply() | Call to the obsolete builtin function 'apply'. |
```

Comment:
I think now that you have split the tests into `2` and `3`, there's no need for this general one. (Indeed, it can't work, as it'll always fail for either Python 2 or 3, as they are bound to give different results.)

Reply:
Done!